hub/api/dataset.py

```diff
@@ -14,7 +14,7 @@
 from typing import Union, Dict, Optional
 import numpy as np
 import warnings
-import os
+from hub.api.pytorch import _to_pytorch
```

Comment:
functions beginning with `_` shouldn't leave the file they are created in. i recommend naming this method `dataset_to_pytorch`